pkg/controllers/clusters/clusterstatus/clusterstatus_controller.go

```diff
@@ -332,3 +342,31 @@ func getNodeCondition(status *corev1.NodeStatus, conditionType corev1.NodeCondit
 	}
 	return -1, nil
 }
+
+// getCommonNodeLabels return the common labels from nodes meta.label
+func getCommonNodeLabels(nodes []*corev1.Node) map[string]string {
+	if len(nodes) == 0 {
```

Comment:
We can simplify this piece of code to avoid unnecessary indents.
```go
if len(nodes) == 0 {
    return nil
}
if len(nodes) == 1 {
    return nodes[0].Labels
}

...
```